tests/e2e/default/enketo/edit-report-with-attachment.wdio-spec.js

```diff
@@ -94,19 +94,19 @@ describe('Edit report with attachmnet', () => {
 
     await commonElements.goToReports();
     await reportsPage.editReport(reportDoc._id);
-    await (await genericForm.fieldByName(formDoc.internalId, 'intro')).setValue('updated text');
+    await (await genericForm.fieldByName(formDoc.internalId, 'intro')).addValue(' updated');
```

Comment:
`setValue` first empties the value and then adds another value. For a brief moment, a validation error for the field was displayed which prevented form submission. 
I suppose I could add the "wait for validation errors" workaround to `submitForm` as well ....